packages/jsActions/nanoflow-actions-native/package.json

```diff
@@ -34,4 +34,4 @@
     "eslint": "^7.20.0",
     "rollup": "~2.66.1"
   }
-}
\ No newline at end of file
+}
```

Comment:
@jordanch @Ahmad-Elsayed Any idea why this is coming as a change 🤔

Reply:
It's whitespace

Reply:
![Screenshot 2022-06-28 at 09 21 34](https://user-images.githubusercontent.com/6040547/176118335-9b5c533a-39a2-4aff-ac87-cb8d4a93a2bb.png)

Reply:
applying 'hide whitespace' shows diff without this line

Reply:
add a new line at the end of the file

Reply:
@jordanch Thanks for the info. That works.

Reply:
@Ahmad-Elsayed Do I need to add a new line 🤔? It's not present in master.

Reply:
I checked the code on my machine, the new line is already added. This the change git is presenting.

Reply:
This is confusing, I don't see the changes on master.

Reply:
I think the PR is good to go, no need for changes.